.redhat-ci.yml

```diff
@@ -8,8 +8,14 @@ required: true
 container:
     image: projectatomic/ostree-tester
 
+packages:
+  - libasan
+  - libubsan
```

Comment:
Can we add `libasan` to the Dockerfile as well? `libubsan` should already be in the image, so there shouldn't be any need to have it here.